source/extensions/quic_listeners/quiche/platform/quic_logging_impl.h

```diff
@@ -179,4 +180,19 @@ class QuicLogSink {
 // Return the previous sink.
 QuicLogSink* SetLogSink(QuicLogSink* new_sink);
 
+// Overload std::operator<< to output a vector.
+template <class T> std::ostream& operator<<(std::ostream& out, const std::vector<T>& v) {
```

Comment:
This function is not specific to QUIC, can you move it to a better place?

Reply:
Only quic use this type of logging right? Envoy has it's only logging macros.

Reply:
It doesn't need to be used by the QUIC_LOG macros. It can also be used in unit tests to pretty-print the content of a vector, similar to https://github.com/envoyproxy/envoy/blob/e0c92845b71e7bb52a6be56727683932c013fa48/test/common/network/cidr_range_test.cc#L22

Reply:
source/common/common/stl_helpers.h sounds like a fine place for this.

Reply:
done